fix build errors with newer binutils:
../sysdeps/unix/syscall-template.S: Assembler messages:
../sysdeps/unix/syscall-template.S:82: Error: CFI instruction used without previous .cfi_startproc
../sysdeps/unix/syscall-template.S:82: Error: CFI instruction used without previous .cfi_startproc
../sysdeps/unix/syscall-template.S:82: Error: CFI instruction used without previous .cfi_startproc
../sysdeps/unix/syscall-template.S:82: Error: CFI instruction used without previous .cfi_startproc
make[2]: *** [build/misc/select.o] Error 1

--- a/ports/sysdeps/unix/sysv/linux/arm/nptl/sysdep-cancel.h
+++ b/ports/sysdeps/unix/sysv/linux/arm/nptl/sysdep-cancel.h
@@ -28,12 +28,10 @@
 # define PSEUDO(name, syscall_name, args)				\
   .section ".text";							\
     PSEUDO_PROLOGUE;							\
-  .type __##syscall_name##_nocancel,%function;				\
-  .globl __##syscall_name##_nocancel;					\
-  __##syscall_name##_nocancel:						\
+  ENTRY (__##syscall_name##_nocancel);					\
     DO_CALL (syscall_name, args);					\
     PSEUDO_RET;								\
-  .size __##syscall_name##_nocancel,.-__##syscall_name##_nocancel;	\
+  END (__##syscall_name##_nocancel);					\
   ENTRY (name);								\
     SINGLE_THREAD_P;							\
     DOARGS_##args;							\
